packages/mui-material/src/Select/SelectInput.js

```diff
@@ -497,7 +497,7 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         ref={handleDisplayRef}
         tabIndex={tabIndex}
         role="combobox"
-        aria-controls={listboxId}
+        aria-controls={open ? listboxId : ''}
```

Comment:
```suggestion
        aria-controls={open ? listboxId : undefined}
```

Thanks for working on this @siddhantantil39 ~

Though an empty string is valid, I think its better to let it be `undefined` to keep an empty `aria-controls` out of the DOM:

![Screenshot 2025-01-06 at 12 33 09 PM](https://github.com/user-attachments/assets/28bfa07d-4021-4d00-acfc-db6cf1741a0e)